Project.toml

```diff
@@ -17,7 +17,7 @@ Ipopt = "1"
 JuMP = "1"
 Memento = "1"
 PowerModels = "0.19.8"
-julia = "1.6"
+julia = ">=1.6"
```

Comment:
`"1.6"` is equivalent to `"^1.6"` (since [caret is the default specifier](https://pkgdocs.julialang.org/v1/compatibility/#Caret-specifiers)) which in turn means any version from 1.6 (included) to 2.0.0 (excluded). No need for changing it, then.